modules/navier_stokes/src/executioners/SIMPLE.C

```diff
@@ -0,0 +1,1106 @@
+                                    "The absolute tolerance on the normalized residual in the "
+                                    "linear solver of the energy equation.");
+  params.addRangeCheckedParam<unsigned int>(
+      "energy_l_max_its",
+      10000,
+      "0<energy_l_max_its",
+      "The maximum allowed iterations in the linear solver of the energy equation.");
+  params.addRangeCheckedParam<Real>("solid_energy_l_tol",
+                                    1e-5,
+                                    "0.0<=solid_energy_l_tol & solid_energy_l_tol<1.0",
+                                    "The relative tolerance on the normalized residual in the "
+                                    "linear solver of the solid energy equation.");
+  params.addRangeCheckedParam<Real>("solid_energy_l_abs_tol",
+                                    1e-10,
+                                    "0.0<solid_energy_l_abs_tol",
+                                    "The absolute tolerance on the normalized residual in the "
+                                    "linear solver of the solid energy equation.");
+  params.addRangeCheckedParam<unsigned int>(
+      "solid_energy_l_max_its",
+      10000,
+      "0<solid_energy_l_max_its",
+      "The maximum allowed iterations in the linear solver of the solid energy equation.");
+  params.addRangeCheckedParam<Real>("passive_scalar_l_tol",
+                                    1e-5,
+                                    "0.0<=passive_scalar_l_tol & passive_scalar_l_tol<1.0",
+                                    "The relative tolerance on the normalized residual in the "
+                                    "linear solver of the passive scalar equation(s).");
+  params.addRangeCheckedParam<Real>("passive_scalar_l_abs_tol",
+                                    1e-10,
+                                    "0.0<passive_scalar_l_abs_tol",
+                                    "The absolute tolerance on the normalized residual in the "
+                                    "linear solver of the passive scalar equation(s).");
+  params.addParam<unsigned int>(
+      "passive_scalar_l_max_its",
+      10000,
+      "The maximum allowed iterations in the linear solver of the passive scalar equation.");
+  params.addRangeCheckedParam<unsigned int>(
+      "num_iterations",
+      1000,
+      "0<num_iterations",
+      "The number of momentum-pressure-(other fields) iterations needed.");
+
+  params.addParam<bool>(
+      "print_fields",
+      false,
+      "Use this to print the coupling and solution fields and matrices throughout the iteration.");
+
+  params.addParam<bool>(
+      "pin_pressure", false, "If the pressure field needs to be pinned at a point.");
+  params.addParam<Real>(
+      "pressure_pin_value", 0, "The value which needs to be enforced for the pressure.");
+  params.addParam<Point>("pressure_pin_point", "The point where the pressure needs to be pinned.");
+
```

Comment:
add param groups for linear solve params and for pressure pin, maybe systems too

Reply:
I'd like the user to see the systems without clicking in the group. Added groups for all the others.